src/api/article/index.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+
+import { requireAuth } from 'auth';
+import * as controller from './controllers';
+import Article, { ArticleType } from './models';
+
+const router = Router();
+router.get('/', controller.getAll);
+router.get('/:id', controller.getById);
+
+router.post('/', requireAuth, controller.create);
```

Comment:
should be protected with `requireAuth` and `requireAdminRole` or smthg with same functionality (modify articles can only `admins` and `creators`)